docs/release-notes/1.3.0/release-1.3.0.md

```diff
@@ -0,0 +1,97 @@
+# [ML.NET](http://dot.net/ml) 1.3.0
+
+## **New Features**
+- **Deep Neural Networks Training (PREVIEW)** ([#4057](https://github.com/dotnet/machinelearning/pull/4057))  
```

Comment:
I think we should be clearer about what's in 1.3 release and what's being released as PREVIEW in 0.15 which is not really 1.3.

In this case, DNN retrain with transfer learning is PREVIEW as part of 0.15 Preview, not really part of 1.3.0 packages, so probably in the 1.13 RELEASE notes we should start with what's in the GA/RTM release which in this case is mostly about bug fixing.

Reply:
@CESARDELATORRE I have put "PREVIEW" in heading that implies everything is preview, no?

Reply:
I get that, but feels strange that the title is about the 1.3 release and it starts with PREVIEW features that are NOT part of 1.3 but part of PREVIEW 0.15.

If I read this as a customer without knowing about the existance of 0.15 PREVIEW I would thinkg that the features DNN, DatabaseLoader, etc. are actually part of 1.3 but somehow in PREVIEW? It is simply confusing because in reality, those features in PREVIEW will be part of upcoming GA releases, such as 1.4 or 1.5 depending on when we release it as GA.

Just saying that  we should be clearer on what's exactly coming as 1.3 release (GA), mostly bug fixing, then we can say that in the same timeframe we're releasing features in PREVIEW versioned as 0.15 packages. That's what we are really doing, right?

Reply:
You seem to be overthinking. Lets keep things simple, "its a 1.3 release YAY! and hey we have few preview features in these, these and these packages" ... if they search for those packages they will see ONLY preview version. Preview means Preview ... its not release, it can change anytime.